### PR TITLE
Add volley dependency

### PR DESCRIPTION
**Pull request by padmaprabha-v**
use latest support library version

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
```
Comment (rama56) [inline app/build.gradle:4]: Where is the change to add volley? What should I review here? :P

File: app/build.gradle [hunk 2]
```diff
@@ -18,10 +18,14 @@ android {
...
+}
 
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
     testCompile 'junit:junit:4.12'
-    compile 'com.android.support:appcompat-v7:23.4.0'
-    compile 'com.google.android.gms:play-services-maps:9.2.1'
+    compile "com.android.support:appcompat-v7:${supportLibVersion}"
+    compile 'com.google.android.gms:play-services-maps:11.0.1'
+    compile 'com.android.volley:volley:1.0.0'
```
Comment (rama56) [inline app/build.gradle:30]: This is the only addition?

Comment (padmaprabha-v) [reply app/build.gradle:30]: Yes

File: app/build.gradle (hunk 1 above)
Comment (padmaprabha-v) [reply app/build.gradle:4]: This was made to use the latest support library

**Review by rama56 — APPROVED** [review]

**Pull request merged**